dask/config.py

```diff
@@ -448,7 +448,7 @@ def rename(aliases, config=config):
             new[n] = value
 
     for k in old:
-        del config[k]  # TODO: support nested keys
+        del config[canonical_name(k,config)]  # TODO: support nested keys
```

Comment:
I suspect also that the linter will yell at you here.  It'll want a space after the comma.

We use flake8 to lint our code.  You can check this locally by running the following from the root directory

```
pip install flake8
flake8 dask
```

Reply:
ah cool I was just getting to installing flake8 and figuring out what it was. Have added the space and run it and it passes (though it didn't seem to raise any issues without the space either)